jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -161,14 +205,16 @@ private T call(RequestState requestState) throws IOException, RegistryException
         }
       }
 
+    } catch (HttpHostConnectException | SSLPeerUnverifiedException ex) {
+      // Tries to call with HTTP protocol if HTTPS failed to connect.
+      if (DEFAULT_PROTOCOL.equals(requestState.url.getProtocol())) {
```

Comment:
I feel like we should explicitly compare against "https" instead of `DEFAULT_PROTOCOL` for readability (and also to avoid problems on the off chance we change the default), or rename the constant. Since we're assuming `DEFAULT_PROTOCOL` is https here.

Reply:
Good point, will change to that.